packages/server/src/auth/utils/cookies.ts

```diff
@@ -52,11 +55,11 @@ export function setAuthCookies(response: GenericResponse, authMaterial: AuthMate
     responseAdapter.setCookies(cookies);
 }
 
-function getCookieHeader(request: GenericRequest): string {
+function getCookieHeader(request: GenericRequest, throwError = true): string | null | undefined {
```

Comment:
Do we need to add both `| null | undefined`?
What does `const cookieHeader = requestAdapter.getCookieHeader()` return?

Reply:
the fetch adapters can return undefined and the node adapter can return null hahaha but i'll move the undefined to force a null